index.html

```diff
@@ -1,140 +1,170 @@
-						<div>
-							<img src="img/details_features.png" alt="BASH Drawing">
-							<img src="img/details_contribute.png" alt="Git for Windows Logo">
-						</div>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<h2>Tools &amp; Features</h2>
+			<p class="text-muted mb-5">Git for Windows focuses on offering a lightweight, native set of tools that bring
+				the full feature set of the <a href="https://git-scm.com" target="_blank">Git SCM</a> to Windows while
+				providing appropriate user interfaces for experienced Git users and novices alike.</p>
+			<div class="row">
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw1web_thumb.png" alt="Git BASH" class="img-other-logos">
+					<h3>Git BASH</h3>
+					<p>Git for Windows provides a BASH emulation used to run Git from the command line. *NIX users
+						should feel right at home, as the BASH emulation behaves just like the "git" command in LINUX
+						and UNIX environments.</p>
+				</div>
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw2web_thumb.png" alt="Git GUI" class="img-other-logos">
+					<h3>Git GUI</h3>
+					<p>As Windows users commonly expect graphical user interfaces, Git for Windows also provides the Git
+						GUI, a powerful alternative to Git BASH, offering a graphical version of just about every Git
+						command line function, as well as comprehensive visual diff tools.</p>
+				</div>
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw3web_thumb.png" alt="Gitk: Exploring Commits" class="img-other-logos">
+					<h3>Shell Integration</h3>
+					<p>Simply right-click on a folder in Windows Explorer to access the BASH or GUI.</p>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="bg-light" id="contribute">
+		<div class="container">
+			<h2>Contribute</h2>
+			<p>As an open source project, Git for Windows benefits greatly from both the volunteer work of helpful
+				developers and <a href="https://github.com/git-for-windows/git/wiki/Issue-reporting-guidelines">good bug
+					reports</a> made by Git for Windows' users.</p>
+			<h3>Governance Model</h3>
+			<p>If you are interested in contributing to the <i>Git for Windows</i> project, have a look at the current
+				<a href="governance-model.html">governance model</a>. There the roles of those involved in driving this
+				project further down the road are described in detail.</p>
+			<h3><a name="issues"></a>Bug Reports &amp; Feature Requests</h3>
+			<p>If you've noticed a bug or simply have an idea that you'd like to see become real, why not work on it?
+				Bug reports and feature requests are typically submitted to the issue tracker on our GitHub repository.
+			</p>
+			<a class="btn btn-primary shadow mr-2" href="https://github.com/git-for-windows/git/issues"
+				target="_blank">Issue Tracker</a>
+			<br>
+			<img src="img/details_contribute.png" class="rounded float-left img-git-logo" alt="Git for Windows Logo">
```

Comment:
The "git SDK" image here looks.. weird. The drop shadow on the text almost has a blue tinge to it?

Reply:
Can't 👍 this one enough. Can we get a render of the Git SDK logo that's full-alpha so that it works on any background, maybe?